docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md

```diff
@@ -31,6 +31,18 @@ This guide assumes you're *up to date with NorthstarProton*, as the method used
 If you're using an older version of NorthstarProton, replace the information about launch options with a text file called `run_northstar.txt` in your Titanfall2 directory. This text file should have a single character `1` inside.
 {% endhint %}
 
+### Ubuntu / Debian
+This workaround employs the use of [Distrobox](https://github.com/89luca89/distrobox/tree/main) to create a Fedora instance and install Steam to it. Your Fedora instance created through this method will share the same Steam files that the native package uses, so you won't have to re-download Titanfall 2.
+
+1. Open a terminal and install Distrobox with `sudo apt-get install distrobox`
+2. Run `distrobox-create --image fedora:latest --name Fedora`
+3. Enter your newly created instance with `distrobox-enter Fedora`
+4. Enable `rpm-fusion` by running `sudo dnf install https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm`
+5. Install Steam by running `sudo dnf install steam`
+6. Export Steam out of your instance by running `distrobox-export steam`, this creates a `.desktop` file outside of the instance that allows Steam to be ran in the container from a shortcut, allowing you to run it without needing to enter your Fedora instance every time.
+7. Search for `Steam (Runtime) (on Fedora)` in your Applications, and optionally pin it to your taskbar.
+8. If you haven't already, follow the steps above to install NorthstarProton.
+
```

Comment:
Maybe also mention how to maintain distrobox container (updates etc)

Reply:
that feels a bit out of the scope of the guide, since it would just be running `dnf update` inside the container, and shouldn't necessarily need it since its just a fire & forget workaround, but i'll add it anyway

Reply:
latest commit resolves this

Reply:
Ok the `systemd` service is good to know but definitely overkill xD

I was thinking more like just telling user how to run `dnf update` in distrobox container ^^"

Reply:
That's there, but I figured some people would like to set up automatic updates, so while it's a lot of text imo it's still worth having

Reply:
latest commit resolves